calibration/experiments/sphere_held_suarez_rhoe_equilmoist/prior.toml

```diff
@@ -0,0 +1,5 @@
+["equator_pole_temperature_gradient_wet"]
+prior = "Parameterized(Normal(4.779568,0.31223328))"
+constraint = "[bounded_below(0)]"
+# EKP.ParameterDistributions.constrained_gaussian("name", 125, 40, 0, Inf)
```

Comment:
quick note - we have the funtionality to create a `constrained_gaussian` directly in EKP in v1.1.5 I thought so why is this still done with in the unconstrained format?

Reply:
I think I created this prior.toml before adding the `constrained_gaussian` parser. I'll update it now, thanks!